apps/dotcom/sync-worker/src/TLUserDurableObject.ts

```diff
@@ -46,12 +46,11 @@ export class TLUserDurableObject extends DurableObject<Environment> {
 			this.env.TL_PG_REPLICATOR.idFromName('0')
```

Comment:
Let's also use the helper here. I have a PR that will help with pinning things closer to db, so it will also help with that.